lib/broadway.ex

```diff
@@ -948,20 +963,23 @@ defmodule Broadway do
 
   defp test_messages(broadway, data, batch_mode, opts) do
     metadata = Map.new(Keyword.get(opts, :metadata, []))
+    acknowledger = Keyword.get(opts, :acknowledger)
```

Comment:
```suggestion
    acknowledger = Keyword.get(opts, :acknowledger, fn _, ref -> {Broadway.CallerAcknowledger, {self(), ref}, :ok} end)
```

This way we don't need `make_ack`? 🤔

Reply:
I initially did not want to do it to avoid an unneeded function call, but I just realized I've added another function definition and call over that.. I will apply this.